staging/src/k8s.io/client-go/examples/leader-election/main.go

```diff
@@ -62,15 +59,11 @@ func main() {
 	var id string
 
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
-	flag.StringVar(&id, "id", "", "the holder identity name")
+	flag.StringVar(&id, "id", uuid.New().String(), "the holder identity name")
 	flag.StringVar(&leaseLockName, "lease-lock-name", "example", "the lease lock resource name")
```

Comment:
I would drop the default lease lock name and namespace and require values be passed in via args. Assuming people will copy this verbatim, we don't want lots of random controllers competing on the example/default name/namespace

Reply:
done